src/common/Header.jsx

```diff
@@ -17,11 +17,16 @@ export const logOutHandler = e => {
   logout()
 }
 
+export const getLink = () => {
+  if(getKeycloak().authenticated) return '/filing/institutions'
+  return '/filing'
```

Comment:
We should add the trailing slash to this `return`.

```suggestion
  return '/filing/'
```

_It's not required for other paths, just the root._